develop/application-devs/interact/endpoints.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Node Endpoints
+description: List of node endpoints and examples of use.  These endpoints can be used for development, RPC, or other purposes in which network access is required.
+---
+
+Ideally, one may run their own node when interacting with the
+[Polkadot network](https://polkadot.network/) via [Polkadot-JS Apps](https://polkadot.js.org/apps/)
+or other UIs and programmatic methods. Another option would be to connect to one of the several
+public endpoints provided by infrastructure and API service providers. For development convenience,
+[Parity Tech](https://www.parity.io/) maintains archive nodes for Polkadot, Kusama, and their test
```

Comment:
It seems that there are multiple line breaks in this paragraph. Could you please review the formatting of this paragraph as well as the others?

Reply:
Yes, comes from copying from the Wiki which has a specific formatter. WIll review and ensure formatting is normal